stream-chat-android-client/src/main/java/io/getstream/chat/android/client/notifications/handler/ChatNotificationHandler.kt

```diff
@@ -272,6 +273,17 @@ public open class ChatNotificationHandler @JvmOverloads constructor(
             ?.asyncGenerateDevice(onDeviceCreated)
     }
 
+    private fun Channel.getNotificationContentTitle(): String =
+        name.takeIf { it.isNotEmpty() }
+            ?: getMemberNamesWithoutCurrentUser()
+            ?: context.getString(R.string.stream_chat_notification_title)
+
+    private fun Channel.getMemberNamesWithoutCurrentUser(): String? = members.filter {
+        it.getUserId() != ChatClient.instance().getCurrentUser()?.id ?: ""
```

Comment:
We duplicate some logic from `Channel.getUsers` here, which lives in `ui-common`. Perhaps we should move it to a lower layer (and rename, as it has a quite confusing name). Assuming that we can make it depend on `ChatClient` instead of `ChatDomain`.